File: FUNCIONAL(Haskell)/CinemaInfo.hs
module CinemaInfo where

-- Função para obter o endereço do cinema
getAddress :: String
getAddress = "R. Irineu Joffily, 23-65 - Centro, Campina Grande. Cine Capitólio"

-- Função para obter o contato do cinema
getContact :: String
getContact = "4002-8922"

-- Função para obter o horário de funcionamento
getOperatingHours :: String
getOperatingHours = "10h00 - 00h00"

-- Função para exibir todas as informações do cinema
viewCinemaInfo :: IO ()
viewCinemaInfo = do
  putStrLn "Informações do Cinema:"
  putStrLn $ "Endereço: " ++ getAddress
  putStrLn $ "Contato: " ++ getContact
  putStrLn $ "Horário de Funcionamento: " ++ getOperatingHours

File: LOGICO(Prolog)/CinemaInfo.pl
% Função para obter o endereço do cinema
get_address("R. Irineu Joffily, 23-65 - Centro, Campina Grande. Cine Capitolio").

% Função para obter o contato do cinema
get_contact("4002-8922").

% Função para obter o horário de funcionamento
get_operating_hours("10h00 - 00h00").

% Função para exibir todas as informações do cinema
view_cinema_info :-
    get_address(Address),
    get_contact(Contact),
    get_operating_hours(OperatingHours),
    write("Informacoes do Cinema:"), nl,
    write("Endereco: "), write(Address), nl,
    write("Contato: "), write(Contact), nl,
    write("Horario de Funcionamento: "), write(OperatingHours), nl.

